docs/extensions/migrations.md

```diff
@@ -19,12 +19,13 @@ for example:
 
 ## Structure
 
-Migrations have to export an `up` and a `down` function. These functions get a [Knex](http://knexjs.org) instance that
-can be used to do virtually whatever.
+Migrations have to export an `up` and a `down` function. These functions get a [Knex](http://knexjs.org) instance, that
+can be used to do virtually whatever, and optional logging parameters.
 
 ```js
 module.exports = {
-	async up(knex) {
+	async up(knex, { logger, log }) {
+		if (log) logger.warn('Creating test table');
```

Comment:
Instead of having a boolean flag `log` and then conditionals, should we instead maybe simply make `logger` either optional or a no-op? 🤔 

We could provide logger as `const logger = { warn: () => {} }` when log is disabled from above

Reply:
That's a good point. I suppose the question is; What is the intention of `log`? I was thinking of it as more of a verbose flag but if it is meant to disable all logging then that might be the way to go.

Reply:
I think it should enable/disable all logging within the migrations. You can already use the LOG_LEVEL setting to only print certain log lines based on severity, so those two combined means you can really fine-tune what you'll get to see from the migrations logs 🙂